### PR TITLE
docs: add instructions to customize font family for TW

### PR DESCRIPTION
**Pull request by RobinCsl**
# This Pull Request meets the following criteria:

- [ ] Tests have been added/adjusted for my new feature
- [ ] New Components are registered in index.js of my project
- [ ] New Components has both `.flow` and `d.ts` files and are exported in `index.js.flow` and `index.d.ts`

 Storybook: https://orbit-mainframev-rcsl-font-instructions.surge.sh

### CODE REVIEW

File: docs/src/documentation/05-development/04-migration-guides/03-v10.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ Components in Tailwind should have almost the same API as before, so few breakin
 As mentioned before, this version of Orbit includes the first batch of components styled with Tailwind. Therefore, Tailwind must be installed and configured in your project for you to be able to use these components.
 Please refer to the [official Tailwind documentation](https://tailwindcss.com/docs/installation) and the [Orbit's Tailwind setup documentation](/foundation/tailwind/installation) for more information on how to do that.
 
-As a summary, you should have Tailwind installed and configured in your project, as well as the Orbit preset for Tailwind included on the config file.
+As a summary, you should have Tailwind installed and configured in your project, as well as the Orbit preset for Tailwind included on the config file. Make sure to extend the theme to override Roboto as the default font family if you need.
```
Comment (mainframev) [inline docs/src/documentation/05-development/04-migration-guides/03-v10.mdx:24]: Not blocking, just suggestion 

```suggestion
In summary, your project should have Tailwind installed and configured, with the Orbit preset for Tailwind included in the configuration file. If necessary, be sure to extend the theme to customize the default font family and replace Roboto.
```

**Review by mainframev — COMMENTED** [review]
LGTM, small suggestion only

**Review by mainframev — APPROVED** [review]

**Comment by mvidalgarcia** [comment]
@RobinCsl Totally unrelated, but since you're updating files at the same level, could you please delete [this leftover file](https://github.com/kiwicom/orbit/blob/master/docs/src/documentation/02-foundation/11-tailwind/03-status.mdx)?

**Review by mvidalgarcia — APPROVED** [review]
LGTM in any case!

**Pull request merged**